clients/python/README.md

```diff
@@ -1,24 +1,79 @@
-# Unity Catalog Python Client
+# Unity Catalog Python Client SDK
 
-The Python SDK for Unity Catalog that is published to https://pypi.org/project/unitycatalog/.
+The Python SDK for Unity Catalog that is published to https://pypi.org/project/unitycatalog-client/.
 
 ## Generate Client Library
 
 To generate the client library code from the API specification in `api/all.yaml`, run:
+
 ```sh
 build/sbt pythonClient/generate
 ```
 
 The generated library will be located in `clients/python/target`.
 
+## Package Build Process
+
+The sbt build configuration that runs to build a deployable package for the SDK
+replaces several generic auto-generated files and performs some required cleanup actions
+on the generated directory structure to ensure that this package conforms to the
+namespace package configuration required by [this guidance](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/).
+
+This package shares the root `unitycatalog.*` namespace with the other suite of
+unitycatalog official packages (`unitycatalog-ai`, `unitycatalog-openai`, etc.).
+
+The sbt assembly process for this SDK will:
+
+1. Remove the generated `__init__.py` file from within the root `unitycatalog` directory. Having any `__init__.py` file in any of the shared namespace packages
+at this level will invalidate the inferred shared namespace logic when Python
+resolves common namespaces, effectively breaking all installed packages.
+2. Remove the `git_push.sh` script as it is not needed and should not be used for
```

Comment:
why are we manually removing this file. why not add it to the open api ignore file and not generate it.